src/java.base/linux/classes/jdk/internal/platform/cgroupv1/CgroupV1Subsystem.java

```diff
@@ -394,6 +408,12 @@ public long getMemorySoftLimit() {
         return CgroupV1SubsystemController.longValOrUnlimited(getLongValue(memory, "memory.soft_limit_in_bytes"));
     }
 
+    /*****************************************************************
+     *  pids subsystem
+     ****************************************************************/
+    public long getPidsMax() {
+        return CgroupV1SubsystemController.longValOrUnlimited(getLongValue(pids, "pids.max"));
```

Comment:
Since this value may be `max` we should use the same logic than for v2. I.e.:

```
 String pidsMaxStr = CgroupSubsystemController.getStringValue(pids, "pids.max");
 return CgroupSubsystemController.limitFromString(pidsMaxStr);
```